open_humans/account_views.py

```diff
@@ -141,3 +162,112 @@ def delete(self, request, *args, **kwargs):
 
     def get_object(self, queryset=None):
         return self.request.user
+
+
+class ResetPasswordView(AllauthPasswordResetView):
+    """
+    Ooops, we've done lost our password, Martha!
```

Comment:
Hah. But, before wrapping this up, I'd like to check that docstrings are being used in a way that helps us later -- to help us and others understand the code that's here in the future. The sorts of questions I raised during review, anticipating what another developer (or "future you", hah) might wonder about. A standard approach: why a subclass was created (or _wasn't_ used - if it was redone from scratch that's very noteworthy), what it's doing differently. :)

e.g. `"""Subclass allauth to use a custom form that preserves a "next" parameter"""`